cache/cache-core/pom.xml

```diff
@@ -82,6 +82,12 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.11.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
What is the reason for this?

Reply:
Ah, that makes sense now that I think about it. I removed commons-io from RESTEasy.